tasties_app/templates/tasties_app/recipes.html

```diff
@@ -5,6 +5,22 @@
     <link rel="stylesheet" type="text/css" href="{% static 'css/recipes.css' %}" />
 {% endblock %}
 {% block content %}
+    {% block categories_block %}
+        <div class="container-fluid bg-light d-flex justify-content-center" id="category_btn_container">
+            <form method="get" action="{% url 'recipes' %}">
+                <button type="submit" class="btn" name="category" value="remove_filter" id="remove_filter">Remove Filter</button>
+            </form>
+            <form method="get" action="{% url 'recipes' %}"  class="form_button">
+                <ul class="list-group list-group-horizontal list-unstyled" id="ul_categories">
+                {% for i in categories_list %}
+                    <li class="list-group-item-sm">
+                        <button type="submit" class="btn" name="category" value="{{i}}" id="category_btn">{{i}}</button>
+                    </li>
+                {% endfor %}
+                </ul>
+            </form>
+        </div>
+    {% endblock categories_block %}
     <div class="grid">
     <div class="row">
     {% for recipe, rating in recipes_with_ratings.items %}
```

Comment:
Here we will completely remove `recipes_with_ratings.items`, in fact the line itself should be:
`{% for recipe, rating in recipes_list %}`